shared/menubar/index.js

```diff
@@ -50,7 +50,7 @@ class Menubar extends Component<void, Props, void> {
       })
     }
 
-    if (module.hot) {
+    if (typeof module.hot === 'function') {
```

Comment:
I believe `module.hot` should be an object at runtime. Just checked and it appears so.